README.md

```diff
@@ -70,7 +55,7 @@ In case of questions or issues with Clairvoyance please refer to [wiki](https://
 
 ## Contributing
 
-Pull requests are welcome! For major changes, please open an issue first to discuss what you would like to change. For more information about tests, internal project structure and so on refer to [Development](https://github.com/nikitastupin/clairvoyance/wiki/Development) wiki page.
+Check our [contributing guide](.github/CONTRIBUTING.md).
```

Comment:
I like the original wording more as it provides a brief for those who don't want to dive deep into the guidelines (so they can go an open an issue). At the same time it does reference the guidelines so that those who are more dedicated can go and get necessary information. What do you think?

Reply:
I will respond below